CHANGELOG.md

```diff
@@ -95,8 +109,6 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Environment Manager Sensu Alerts for Node
 - [Truncation of service version identifiers at 50 characters in the UI](https://gitlab.tools.trainline.eu/capitainetrain/mweb/issues/487). Service version identifiers now limited to 100 characters in UI and API. User notified if a longer version identifier is pasted into UI. [#347]
 
-## Unreleased
```

Comment:
Ok I've got to know... Why do you delete this? Isn't it helpful to know what changes master contains that have not yet been included in a release?